cli/src/format.rs

```diff
@@ -295,6 +295,7 @@ where
         PackageType::PyPi => "PyPI",
         PackageType::Maven => "Maven",
         PackageType::Nuget => "NuGet",
+        _ => "unknown",
```

Comment:
I assume this is for `Golang`? I'd rather be explicit here and put a `PackageType::Goland => unimplemented!()` until we merge #720, I wouldn't want to risk involuntarily incomplete matches in the future.

Reply:
It's just that our own `LockfileFormat` is `non_exhaustive`, so I thought this might be too or there was a PR up to make some part of phylum-types non exhaustive. But I can't find anything.